imod/prepare/wells.py

```diff
@@ -47,6 +47,7 @@ def locate_wells(
     top: Union[xr.DataArray, xu.UgridDataArray],
     bottom: Union[xr.DataArray, xu.UgridDataArray],
     k: Union[xr.DataArray, xu.UgridDataArray, None],
+    throw_if_well_not_placed: bool = False,
```

Comment:
Maybe just name this ``validate``? I see further down:

```
    if not np.array_equal(xy_bottom["index"], index):
        raise ValueError("bottom grid does not match top grid")
    if k is not None and not np.array_equal(xy_k["index"], index):
        raise ValueError("k grid does not match top grid")
```

We could set this to:

```
    if validate and not np.array_equal(xy_bottom["index"], index):
        raise ValueError("bottom grid does not match top grid")
    if validate and (k is not None) and not np.array_equal(xy_k["index"], index):
        raise ValueError("k grid does not match top grid")
```

Reply:
done